.github/workflows/scripts/ti_build/tinysh.py

```diff
@@ -142,8 +137,10 @@ def __call__(self, *moreargs: Sequence[str]) -> None:
         for o in OPTIONS_STACK:
             options.update(o)
 
+        print("Command.__call__ args", args, "cmd", cmd, "optoins", options, "overlay", overlay)
```

Comment:
`optoins` typo

Reply:
The print was left in inadvertently actually, but if you're fine with leaving it in, that works for me :) Consolidated a bit, into a single line, and fixed the spelling of options.

Reply:
This is dev tooling, so I'm fine with it.